src/util/SchemaUtil.cpp

```diff
@@ -11,6 +11,8 @@
 namespace nebula {
 namespace graph {
 
+/*static*/ constexpr std::size_t SchemaUtil::kCommentLengthLimit;
```

Comment:
It seems not necessary to define it again

Reply:
Will lead to compile error.